bin/linters.sh

```diff
@@ -82,6 +82,7 @@ go_metalinter() {
         --enable=vet\
         --enable=vetshadow\
         --skip=testdata\
+        --skip=vendor\
         --vendor\
         $PKGS
 }
```

Comment:
Should we add this .bzl file and others like it to the list of files over which we run buildifier? See the bottom of linters.sh.